src/ResourceManager/PowerBIDedicated/Commands.PowerBIDedicated/Models/AzurePowerBIDedicated.cs

```diff
@@ -0,0 +1,84 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Azure.Management.PowerBIDedicated.Models;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.PowerBIDedicated.Models
+{
+    public class AzurePowerBIDedicated
```

Comment:
@selasagi almost all of your cmdlets return this object. I think that you should have a separate object called `PSDedicatedCapacity` that wraps the `DedicatedCapacity` object from your SDK that you return instead. The user doesn't need to have this type that has extra methods that they'll never use. The only thing they need are the properties.

Reply:
thanks. updated